core/tests/puppeteer-acceptance-tests/user-utilities/logged-in-users-utils.ts

```diff
@@ -314,34 +314,32 @@ export class LoggedInUser extends BaseUser {
       showMessage('The 61 Million Children link opens the right page.');
     }
   }
-
   /**
    * Function to click the even those who are in school link
    * in the About Foundation page and check if it opens the right page.
    */
   async clickEvenThoseWhoAreInSchoolLinkInAboutFoundation(): Promise<void> {
     await this.page.waitForSelector(millionsOfContentId);
+    const anchorElementSelector = `${millionsOfContentId} a:nth-child(2)`;
     const buttonText = await this.page.$eval(
-      millionsOfContentId,
-      element => element.getElementsByTagName('a')[1].textContent
+      anchorElementSelector,
+      element => element.textContent
     );
     if (buttonText !== 'even those who are in school') {
       throw new Error(
         'The Even Those Who Are In School button does not exist!'
       );
     }
-    await this.page.$eval(millionsOfContentId, element =>
-      element.getElementsByTagName('a')[1].click()
-    );
-    if (this.page.url() !== evenThoseWhoAreInSchoolUrl) {
-      throw new Error(
-        'The Even Those Who Are In School link does not open ' +
-          'the right page!'
+    try {
```

Comment:
Do you actually need the try/catch block (what behaves differently with and without it)?

Reply:
If I were to switch from `try/catch` structure to `if/else` structure, I would need to adjust the `openExternalPdfLink()` function to return a boolean value. However, I wonder that might not be the best approach as it could result in the loss of specific error messages that the function currently provides when an issue arises. For instance, logging the actual href and comparing it with the expected one. Additionally, it allows the developer to write an appropriate message when everything functions correctly. 
So, I guess `try/catch` is more apt in the case. What do you think?

Reply:
I didn't ask to switch to an if/else structure. I'm asking what the try/catch adds here compared to just doing the thing in the try block directly, since the catch doesn't seem to do anything interesting.

Reply:
Oh! It was indeed making the debugging difficult as it was hiding the earlier part of the stack trace. Since a try block expects a catch block, I am removing the try/catch structure altogether.
Thanks!